.github/workflows/test-build-processes.yml

```diff
@@ -15,11 +15,11 @@ on:
       - '3.[7-9]'
       - '[4-9].[0-9]'
     paths:
-      # Any change to a PHP, CSS, JavaScript, or JSON file should run checks.
-      - '**.css'
-      - '**.js'
-      - '**.json'
-      - '**.php'
+      # Any change to a source PHP, CSS, JavaScript, or JSON file should run checks.
+      - 'src/**.css'
+      - 'src/**.js'
+      - 'src/**.json'
+      - 'src/**.php'
```

Comment:
@johnbillion What do you think about removing the `composer.*` pattern here? The only dependency managed in Composer that is used in the build process is `composer/ca-bundle`. But the build process will never be affected by this without first manually updating the version (it's pinned to an exact version vs. using a semantic range) AND running `grunt certificates:upgrade`.

Otherwise, the only files related to certificates that would affect the build process are the `.pem` and `.crt` files that are in `src/wp-includes/certificates`. It may be worth adding `**/*.pem` and `**/*.crt` to this list of path filters for those scenarios.